lib/kimurai/base_helper.rb

```diff
@@ -12,7 +12,7 @@ def escape_url(url)
     rescue URI::InvalidURIError => e
       URI.parse(URI.escape url).to_s rescue url
     else
-      url
```

Comment:
This is actually not a bug, the method should return string object (`url`) not URI `uri` if url is correct